app/partials/toggles.html

```diff
@@ -2,50 +2,35 @@
 
 <p ng-show="toggles.length == 0">There are no toggles here!</p>
 
-<form>
-<table class="table table-border">
-  <thead>
-    <tr>
-      <th>Toggle name</th>
-      <th>Toggle status</th>
-      <th>Actions</th>
-    </tr>
-  </thead>
-  <tbody ng-repeat="toggle in toggles" class="toggle">
-    <tr>
-      <td><input ng-model="toggle.name" ng-enter="update(toggle)"></td>
-      <td><select ng-model="toggle.status" ng-options="s.value as s.label for s in statuses"></select></td>
-      <td>
+<div class="row header">
+  <div class="span3">Toggle name</div>
+  <div class="span3">Toggle status</div>
+  <div class="Span6">Actions</div>
+</div>
+<div ng-repeat="toggle in toggles" class="toggle" ng-controller="ToggleCtrl">
```

Comment:
ToggleCtrl prevents the $scope from updating the toggles array (preventing deletes). However, it provides the ability to reach the form to validate on a toggle basis.